node/tessel-export.js

```diff
@@ -917,6 +917,8 @@ Tessel.SPI = function(params, port) {
 
   this.chipSelectActive = params.chipSelectActive === 'high' || params.chipSelectActive === 1 ? 1 : 0;
 
+  this.chipSelectDelay = (params.chipSelectDelayUs/1e3) || 0;
```

Comment:
- `1e3` is only 1 character shorter than `1000` :P
- Ideally, `this.chipSelectDelay` and `params.chipSelectDelayUs` would have the same name and value (instead of ms and μs)
- A sub-millisecond parameter to `setTimeout` is impossible: https://nodejs.org/docs/latest/api/timers.html#timers_setinterval_callback_delay_arg 

  > Note: When delay is larger than 2147483647 or less than 1, the delay will be set to 1.